client/src/Components/OrderDetails.elm

```diff
@@ -241,7 +241,158 @@ orderTable ({ lineItems, products } as details) =
                 Just a ->
                     f a
     in
-    div []
+    div [ class "tw:block tw:lg:hidden" ]
+        [ div [ class "order-details-blocks d-sm-none" ] <|
+            List.map productBlock products
+                ++ [ table [ class "table table-striped table-sm" ] [ tableFooter ] ]
+        ]
+
+
+orderTableDesktop : PageData.OrderDetails -> Html msg
```

Comment:
I see a lot of code duplication with `orderTableMobile`. Is this something we could avoid?